### PR TITLE
Fix li charge

### PR DESCRIPTION
**Pull request by fernandohds564**
This PR adds classes in `currinfo` to provide charge at ICTs from Linac, TB and TS together with their transport efficiency.

It is deeply related to lnls-sirius/machine-applications#204

### CODE REVIEW

File: siriuspy/siriuspy/currinfo/main.py [hunk 1]
```diff
@@ -16,7 +22,6 @@
 
 
 # BO Constants
-
 BO_HarmNum = 828
```
Comment (xresende) [inline siriuspy/siriuspy/currinfo/main.py:25]: `pylint(invalid_name)`

File: siriuspy/siriuspy/currinfo/main.py [hunk 2]
```diff
@@ -2,8 +2,14 @@
...
 import time as _time
 from datetime import datetime as _datetime
+import logging as _log
+from copy import deepcopy as _dcopy
+
 import numpy as _np
 from epics import PV as _PV
+import visa
+
+from mathphys.functions import get_namedtuple
```
Comment (xresende) [inline siriuspy/siriuspy/currinfo/main.py:12]: use of  `_get_namedtuple` to keep module namespace API clean.

File: siriuspy/siriuspy/currinfo/main.py [hunk 3]
```diff
@@ -29,7 +34,6 @@
 BO_CURR_THRESHOLD = 0.06
 
 # SI Constants
-
 SI_HarmNum = 864
```
Comment (xresende) [inline siriuspy/siriuspy/currinfo/main.py:37]: `pylint(invalid_name)`

**Comment by xresende** [comment]
I think we should try to keep _siriupsy_ dependencies to a minimum in order to avoid potential installation and package versioning issues of this basic repo. imports like,

- `siriuspy.currinfo.main`: `import visa`
- `siriuspy.pwrsupply.pructrl.pru`: `import EthBrigdeClient` 
- `siriuspy.optics.lattice_survey`: `import pyaccel`
- `siriuspy.optics.lattice_survey`: `import pymodels`,

should either belong to _machine_applications_, _scripts_ or _apsuite_ repo. 

for the first two cases above, objects `ResorceManager` and `EtherBrigdeClient` could be passed respectively to `_ASCurrInfoApp` and `PRU` initializers as arguments, thus avoid imports and moving package dependency to _machine_applications_ which in my view is more appropriate.

`lattice_survey` maybe should be moved to either _scripts_ or _apsuite_ repo...

File: siriuspy/siriuspy/currinfo/main.py (hunk 3 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/currinfo/main.py]: done!

File: siriuspy/siriuspy/currinfo/main.py (hunk 2 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/currinfo/main.py]: done!

File: siriuspy/siriuspy/currinfo/main.py (hunk 1 above)
Comment (fernandohds564) [reply siriuspy/siriuspy/currinfo/main.py]: done!

**Pull request merged**